bundles/http/Citrix/DocuSign/73f4e3b3-e2f3-43b3-9ce6-7f12da7b714c.js

```diff
@@ -0,0 +1,660 @@
+                {
+                    "email": `${actionParameters.recipient_Email ?? actionParameters.recipient_Email_Unknown}`,
+                    "name": `${actionParameters.recipient_Name}`,
+                    "recipientId": `${parseInt(moment().format('HHmmss'), 10)}`,//parseInt used to remove the leading zeros
+                }
+            ]
+        })
+    })
+    const json = await respone.json()
+
+    if (!respone.ok) {
+        throw new Error(JSON.stringify(json))
+    }
+    await syncEnvelopes(client, dataStore, ACCOUNT_ID, [actionParameters.user_Id])
+}
+
+async function sendTemplateThreeRecipient({ client, dataStore, integrationParameters, actionParameters }) {
+    const ACCOUNT_ID = integrationParameters.account_Id;
+    let templateRoles = [{ "roleName": `${actionParameters.role_Name_one}`, "name": `${actionParameters.recipient_Name_one}`, "email": `${actionParameters.recipient_Email_one ?? actionParameters.recipient_Email_one_Unknown}` }]
+
+    if (actionParameters.role_Name_two != null && actionParameters.recipient_Name_two != null) {
+        templateRoles = [{ "roleName": `${actionParameters.role_Name_one}`, "name": `${actionParameters.recipient_Name_one}`, "email": `${actionParameters.recipient_Email_one ?? actionParameters.recipient_Email_one_Unknown}`},{ "roleName": `${actionParameters.role_Name_two}`, "name": `${actionParameters.recipient_Name_two}`, "email": `${actionParameters.recipient_Email_two ?? actionParameters.recipient_Email_two_Unknown}` }]
+    }
+    if (actionParameters.role_Name_three != null && actionParameters.recipient_Name_three != null) {
+        templateRoles = [{ "roleName": `${actionParameters.role_Name_one}`, "name": `${actionParameters.recipient_Name_one}`, "email": `${actionParameters.recipient_Email_one ?? actionParameters.recipient_Email_one_Unknown}`},{ "roleName": `${actionParameters.role_Name_two}`, "name": `${actionParameters.recipient_Name_two}`, "email": `${actionParameters.recipient_Email_two ?? actionParameters.recipient_Email_two_Unknown}` },{ "roleName": `${actionParameters.role_Name_three}`, "name": `${actionParameters.recipient_Name_three}`, "email": `${actionParameters.recipient_Email_three ?? actionParameters.recipient_Email_three_Unknown}` }]
+    }
+    console.log(JSON.stringify(templateRoles.toString()))
+    const respone = await client.fetch(`accounts/${ACCOUNT_ID}/envelopes`, {
+        method: "POST",
+        body: JSON.stringify({
+            "emailSubject": `${actionParameters.email_Subject}`,
+            "templateId": `${actionParameters.template_Id}`,
+            "templateRoles": templateRoles,
+            "status": "sent"
+        })
+    })
+    const json = await respone.json()
+
+    if (!respone.ok) {
+        throw new Error(JSON.stringify(json))
+    }
+    await syncEnvelopes(client, dataStore, ACCOUNT_ID, [actionParameters.user_Id])
+}
+
+async function updateRecipient({ client, dataStore, integrationParameters, actionParameters }) {
```

Comment:
Also almost the same as `addRecipient`, only different method. This could also be merged with some additional `if` statement.

Reply:
We have merged updateRecipient and resendEnvelope, as there was only difference of Method.